src/core/Object3D.js

```diff
@@ -297,45 +297,38 @@ class Object3D extends EventDispatcher {
 
 	}
 
-	add( object ) {
+	add( ...objects ) {
 
-		if ( arguments.length > 1 ) {
-
-			for ( let i = 0; i < arguments.length; i ++ ) {
+		objects.forEach( object => {
```

Comment:
@LeviPesin let me know what you think of this refactor. The `Object3D.remove` method could be refactored in the same way.

Reply:
I like this refactoring, but let's see what @mrdoob and @Mugen87 will decide...